lib/shield-custom.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+module.exports = {
+  conventionalcommits: {
+    text: 'Conventional Commits',
+    path: 'https://img.shields.io/badge/Conventional%20Commits-1.0.0-yellow.svg',
+    link: 'https://conventionalcommits.org',
+  },
+  spacemacs: {
+    text: 'Built with Spacemacs',
+    path: 'https://cdn.rawgit.com/syl20bnr/spacemacs/442d025779da2f62fc86c2082703697714db6514/assets/spacemacs-badge.svg',
```

Comment:
isn't rawgit deprecated?

> GitHub repositories that served content through RawGit within the last month will continue to be served until at least October of 2019. URLs for other repositories are no longer being served.
https://rawgit.com/

Reply:
I didn't know it was deprecated, but it is the only badge I found https://github.com/syl20bnr/spacemacs#supporting-spacemacs, any ideas ?

Reply:
this https://raw.githubusercontent.com/syl20bnr/spacemacs/master/assets/spacemacs-badge.svg?sanitize=true seems to work, I'll update the PR